### ISSUE TITLE
Duplicate Entries in Google Calendar

### ISSUE THREAD

**Issue by stud0709** [phw198/OutlookGoogleCalendarSync#339]
Hello everyone, 

v. 2.6.1 as well as 2.6.0: syncing to google calendar results in duplicate entries in some cases. I dont have a clue on what the reason might be. If I delete the "Home Office" Entry on Sep. **18**., the log states:

Deleting 1 Google calendar entries...
**13**.09.2017 13:00 => "Home office"
Done.

Both entries on Sep. 18 are still there, the entry on Sep. 13 as well.

If I delete all this duplicate stuff in Google Calendar and resync, the entries are synchronized only once, but the Sep. 13 entry is duplicated, but Sep. 20 is missing.

Creating 2 Google calendar entries...
13.09.2017 13:00 => "Home office"
19.09.2017 13:00 => "Home office"
Done.

And... nothing special within the log file. 
[OGcalsync.log.txt](https://github.com/phw198/OutlookGoogleCalendarSync/files/1303106/OGcalsync.log.txt)

What can I do?

Regards,

Yuriy

![2017-09-14_15h13_15](https://user-images.githubusercontent.com/31960113/30431653-91bb9d10-995f-11e7-8bce-88e4bfb9eabf.png)

**Comment by phw198** [comment]
I think this is a duplicate of #224 - I suspect you copied one of the Home Office appointments in Outlook to create the other?

I'd suggest upgrading to v2.6.2 and following [the same steps](https://github.com/phw198/OutlookGoogleCalendarSync/issues/314#issuecomment-330092053) I outlined in the other ticket.
